### ISSUE TITLE
Syntax for Remove-HaloTicketBatch

### ISSUE THREAD

**Issue by machine-bit** [homotechsual/HaloAPI#43]
I am struggling to get this working, as path of teh parameter -tickets I have tried every connotaion of array / object and it still fails stating "Remove-HaloTicketBatch requires an array of ticket ids to delete." Anyone used thsi recently? I have got round it using a foreach loop and the Remove-HaloTicket command, but I have a feeling it might be a bug

**Comment by homotechsual** [comment]
Yeah so that's broken. Sorry. It's got the wrong parameter type. You can fix it locally by changing the type in the `Remove-HaloTicketBatch` to `Int64[]` instead of `array[]`. I'll get it fixed tomorrow and we're overdue a release anyway!

**Comment by homotechsual** [comment]
oh, and change `$Actions -is [Array]` to `$Tickets -is [Array]` this one's a bad copypasta :-D

**Comment by machine-bit** [comment]
@homotechsual I did actually see that bit and wondered what it was doing, it all makes sense now. Thought it was me rather than the code :-)

**Comment by homotechsual** [comment]
Just released 1.21.0 which should fix this :-)

**Comment by machine-bit** [comment]
@homotechsual can lslightly further;
Exception: Exception calling "ShouldProcess" with "2" argument(s): "A command that prompts the user failed because the host program or the command type does not support user interaction. The host was attempting to request confirmation with the following message: Are you sure you want to perform this action?

Is this because Remove-HaloTicket needs the -confirm:$false passing?

**Comment by homotechsual** [comment]
Yeah, it would.
